packages/env/lib/commands/run.js

```diff
@@ -18,41 +19,48 @@ const initConfig = require( '../init-config' );
  * @param {Object}   options
  * @param {string}   options.container The Docker container to run the command on.
  * @param {string[]} options.command   The command to run.
+ * @param {string}   options.envCwd    The working directory for the command to be executed from.
  * @param {Object}   options.spinner   A CLI spinner which indicates progress.
  * @param {boolean}  options.debug     True if debug mode is enabled.
  */
-module.exports = async function run( { container, command, spinner, debug } ) {
+module.exports = async function run( {
+	container,
+	command,
+	envCwd,
+	spinner,
+	debug,
+} ) {
 	const config = await initConfig( { spinner, debug } );
 
 	command = command.join( ' ' );
 
 	// Shows a contextual tip for the given command.
 	showCommandTips( command, container, spinner );
 
-	await spawnCommandDirectly( {
-		container,
-		command,
-		spinner,
-		config,
-	} );
+	await spawnCommandDirectly( config, container, command, envCwd, spinner );
 
 	spinner.text = `Ran \`${ command }\` in '${ container }'.`;
 };
 
 /**
  * Runs an arbitrary command on the given Docker container.
  *
- * @param {Object}   options
- * @param {string}   options.container The Docker container to run the command on.
- * @param {string}   options.command   The command to run.
- * @param {WPConfig} options.config    The wp-env configuration.
- * @param {Object}   options.spinner   A CLI spinner which indicates progress.
+ * @param {WPConfig} config    The wp-env configuration.
+ * @param {string}   container The Docker container to run the command on.
+ * @param {string}   command   The command to run.
+ * @param {string}   envCwd    The working directory for the command to be executed from.
+ * @param {Object}   spinner   A CLI spinner which indicates progress.
  */
-function spawnCommandDirectly( { container, command, config, spinner } ) {
+function spawnCommandDirectly( config, container, command, envCwd, spinner ) {
+	// We need to pass absolute paths to the container.
+	envCwd = path.resolve( '/var/www/html', envCwd );
```

Comment:
Yeah, me neither :) Though, while we're here, could we just support absolute paths as well just in case? 

```suggestion
	envCwd = envCwd.startsWith( "/" ) ? envCwd : path.resolve( '/var/www/html', envCwd );
```

Reply:
this might also be necessary to work with the default value of `/var/www/html`

Reply:
Actually, if you pass an absolute path as a successive argument to `path.resolve()`, it will only use the last path! See: `npm run env -- run tests-wordpress --env-cwd="/var" pwd`

Reply:
Sick!